extension/js/common/platform/xss.ts

```diff
@@ -230,6 +232,15 @@ export class Xss {
     return str.replace(/&/g, '&amp;').replace(/"/g, '&quot;').replace(/'/g, '&#39;').replace(/</g, '&lt;').replace(/>/g, '&gt;').replace(/\//g, '&#x2F;');
   };
 
+  public static stripEmojis = (str: string) => {
+    if (Intl !== undefined && typeof Intl.Segmenter === 'function') {
+      str = [...new Intl.Segmenter().segment(str)].map(x => x?.segment).join('');
+    } else {
+      str = str.replace(Xss.EMOJI_REGEX, '');
+    }
+    return this.escape(str);
+  };
```

Comment:
Sorry for confusion - we don't need to use `Intl.Segmenter` here, as in `emailjs-mime-codec.js` it's used only for separating emojis from other characters (as simple string `split` doesn't work for emojis).

So here it's not needed and this method can be simplified to just:
```js
const stringWithoutEmojis = str.replace(Xss.EMOJI_REGEX, '');
return this.escape(stringWithoutEmojis);
```

Reply:
Ah, yes -- I will remove the usage of `Intl.Segmenter` and will simplify the function as suggested. thanks again.